litmus-portal/frontend/src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import TextField from '@material-ui/core/TextField';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import useStyles from './styles';
+
+// This is the Password Input Component
+interface State {
```

Comment:
The State interface isn't being used anywhere in this component you can get rid of it